tests/commands/test_new.py

```diff
@@ -260,7 +260,9 @@ def test_new_with_flags_edit_set_template(repo: Repo, directory: str) -> None:
     set_values = "mode=set"
```

Comment:
This seems like a leftover from before the rename to `--keys`.

Also, as I cannot comment higher up:
- the docstring is no longer accurate
- the function name should be changed (as you have with others)
  - I won't check all function names; please just search for `set`, etc